docs/guides/transportation.mdx

```diff
@@ -49,6 +63,210 @@ are currently the most developed part of the `transportation` schema. Read more
 
+## Dataset Description
+
+All Overture data, including transportation data, is distributed as GeoParquet, a column-based data structure. Below you'll find a table with column-by-column descriptions of the properties in the transportation feature type.
+
+<details>
+    <summary>Schema for GeoParquet files in the transportation theme</summary>
+<div>
+<Tabs>
+<TabItem value="segment" label="segment" default>
+| column_name | column_type | Description |
+|---|---|---|
+| id | VARCHAR | A feature ID. This may be an ID associated with the Global Entity Reference System (GERS) if—and-only-if the feature represents an entity that is part of GERS. |
+| geometry | BLOB | The line representation of the segment's location. Segment's geometry which MUST be a LineSting as defined by GeoJSON schema. |
+| bbox | STRUCT | Area defined by two longitudes and two latitudes: latitude is a decimal number between -90.0 and 90.0; longitude is a decimal number between -180.0 and 180.0. |
+| version | INTEGER | Version number of the feature, incremented in each Overture release where the geometry or attributes of this feature changed. |
+| sources | STRUCT[] | The array of source information for the properties of a given feature, with each entry being a source object which lists the property in JSON Pointer notation and the dataset that specific value came from. All features must have a root level source which is the default source if a specific property's source is not specified. |
+| subtype | VARCHAR | The broad category of transportation segment. |
+| class | VARCHAR | Captures the kind of road and its position in the road network hierarchy. |
+| names | STRUCT[] | Properties defining the names of a feature. |
+| connector_ids | VARCHAR[] | List of connector IDs identifying the connectors this segment is physically connected to. Each connector is a possible routing decision point, meaning it defines a place along the segment in which there is possibility to transition to other segments which share the same connector. |
+| routes | STRUCT[] | Routes this segment belongs to. |
+| access_restrictions | STRUCT[] | Rules governing access to this road segment or lane. |
+| level_rules | STRUCT[] | Defines the Z-order, i.e. stacking order, of the road segment. |
+| prohibited_transitions | STRUCT[] | Defines where traveling from the segment to another is disallowed for navigation. This covers things situations prohibited turns or a transition from road to bike lane disallowing cars. |
+| road_surface | STRUCT[] | Defines the surface material on a road such as paved, asphalt, or unpaved. |
+| road_flags | STRUCT[] | Additional properties relevant to roads such as is_bridge or is_under_construction. |
+| speed_limits | STRUCT[] | Defines the speed limit of the road segment. |
+| width_rules | STRUCT[] | Defines the width of the road segment for rendering. |
```

Comment:
We're missing connectors, subclass, subclass_rules and destinations in this list. Those will ship for the August release if we want to include them